libs/core/src/lib/bar/button-bar/button-bar.component.ts

```diff
@@ -23,8 +23,8 @@ import { ContentDensityService } from '@fundamental-ngx/core/utils';
               [compact]="compact"
               [fdType]="fdType"
               [label]="label"
-              [attr.title]="title"
-              [attr.aria-label]="ariaLabel"
+              [attr.title]="ariaLabel"
```

Comment:
Shouldn't this first check if the title prop is set and then assign it to ariaLabel? 
We should not take away the option for the user to set the value of title to what they want

Reply:
fixed